### PR TITLE
Remove dependency on plone.directives.form

### PR DESCRIPTION
**Pull request by l34marr**
I guess plone.directives.form can be replaced by z3c.form, plone.supermodel and plone.autoform, so here is my take to remove its dependency. Hopefully this will get rid of grok dependency completely. A fresh manual installation shows it is working.

### CODE REVIEW

**Comment by l34marr** [comment]
I know little about robot tests. Help needed for the remaining test errors.

File: src/collective/cover/behaviors/interfaces.py [hunk 1]
```diff
@@ -25,10 +26,16 @@ class IRefresh(model.Schema):
...
 
-@form.validator(field=IRefresh['ttl'])
-def validate_ttl(value):
-    if value <= 0:
-        raise Invalid(_(u'Value must be greater than zero.'))
+class TimeToLiveValidator(validator.SimpleFieldValidator):
+
+    """z3c.form validator class for time to live"""
+
+    def validate(self, value):
```
Comment (hvelarde) [inline src/collective/cover/behaviors/interfaces.py]: @l34marr this validator is not being tested in any way; can you write a test?

**Comment by hvelarde** [comment]
@idgserpro I need help with this also.

File: src/collective/cover/behaviors/interfaces.py (hunk 1 above)
Comment (idgserpro) [inline src/collective/cover/behaviors/interfaces.py]: @hvelarde it seems to me correct. See: http://docs.plone.org/develop/addons/schema-driven-forms/customising-form-behaviour/validation.html#field-widget-validators

**Comment by hvelarde** [comment]
@idgserpro awesome; I think we can make a new release now, do you agree?

**Comment by idgserpro** [comment]
Yes. Remember bump version to 1.3

**Pull request merged**